baystation12.dme

```diff
@@ -11,14 +11,12 @@
 // END_PREFERENCES
 // BEGIN_INCLUDE
 #include "code\_macros.dm"
-#include "code\client_macros.dm"
 #include "code\global.dm"
 #include "code\hub.dm"
 #include "code\names.dm"
 #include "code\stylesheet.dm"
 #include "code\world.dm"
 #include "code\__defines\_compile_options.dm"
-#include "code\__defines\_planes+layers.dm"
```

Comment:
Check all baystation12.dme changes.
- #include "code\client_macros.dm"
- #include "code\__defines\_planes+layers.dm"
- #include "code\_onclick\hud\parallax.dm"
- etc.
It's out of date.